src/commands/Scheduled.ts

```diff
@@ -0,0 +1,116 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+import { DateTime } from 'luxon';
+import schedule from 'node-schedule';
+import { v4 as newUUID } from 'uuid';
+import Logger from '../utils/Logger';
+
+/**
+ * This command will send a scheduled message after waiting
+ * for a user specified time.
+ * 
+ * 
+ * The command will run a cronjob to send the message. Currently
+ * there is a 1 day limit on scheduled messages but that will likely
+ * change soon.
+ */
+export default class Scheduled extends Command {
+  constructor(client: BotClient) {
+    const definition = new SlashCommandBuilder()
+      .setName('schedulesend')
+      .addStringOption((option) => option.setName('message')
+        .setDescription('message to be sent')
```

Comment:
For consistency, we should keep the descriptions in the same formatting as the description for other commands (e.g. 'Message to be sent', 'Sends a scheduled message to the channel this command is used in.', etc.)